Extension/src/LanguageServer/client.ts

```diff
@@ -1340,7 +1338,7 @@ export class DefaultClient implements Client {
             defaultSystemIncludePath: settings.defaultSystemIncludePath,
             cppFilesExclude: settings.filesExclude,
             clangFormatPath: util.resolveVariables(settings.clangFormatPath, this.AdditionalEnvironment),
-            clangFormatStyle: settings.clangFormatStyle ? util.resolveVariables(settings.clangFormatStyle, this.AdditionalEnvironment) : undefined,
```

Comment:
This appears to be a "breaking" change. If C_Cpp.clang_format_style is "", then it causes the default of "file" to not be used in cpptools.